calico/reference/felix/configuration.mdx

```diff
@@ -198,7 +194,13 @@ See the [HOWTO guide](../../operations/ebpf/enabling-ebpf.mdx) for step-by step
+
 | BPFPolicyDebugEnabled / <br/> FELIX_BPFPOLICYDEBUGENABLED                           | In eBPF dataplane mode, Felix records detailed information about the BPF policy programs, which can be examined with the calico-bpf command-line tool.                                                                                                                                                                                                                            | true, false           | true    |
+| BPFForceTrackPacketsFromIfaces / <br/> FELIX_BPFFORCETRACKPACKETSFROMIFACES         | In BPF mode, forces traffic from these interfaces to skip Calico's iptables NOTRACK rule, allowing traffic from those interfaces to be tracked by Linux conntrack.  Should only be used for interfaces that are not used for the Calico fabric.  For example, a docker bridge device for non-Calico-networked containers. [Default: docker+]                                      | string                | "docker+" |
+| BPFDisableGROForIfaces / <br/> FELIX_BPFDISABLEGROFORIFACES                         | A regular expression that controls which interfaces Felix should disable the Generic Receive Offload [GRO] option. It should not match the workload interfaces (usually named cali...).                                                                                                                                                                                           | string                | ""      |
+| BPFExcludeCIDRsFromNAT / <br/> FELIX_BPFEXCLUDECIDRSFROMNATIS                       | A list of CIDRs that are to be excluded from NAT resolution so that host can handle them. A typical use case is node local DNS cache. Set it to your kube-dns service IP if you want to use node local DNS cache.                                                                                                                                                                  | string                | ""      |
+| BPFRedirectToPeer <br/> FELIX_BPFREDIRECTTOPEER                                     | Controls whether it is allowed to forward straight to the peer side of the workload devices which makes ingress into the workloads from outside the node faster. It is enabled for any host L2 devices by default (`L2Only`), but it breaks tcpdump on the host side of workload device as it bypasses it on ingress. If set it to `Disabled` to make tcpdump work as expected. Value of `Enabled` also allows redirection from L3 host devices like IPIP tunnel or Wireguard directly to the peer side of the workload's device. However, it breaks tools like tcpdump on the peer side. Use Enabled with caution.                                                                                                                                                                                                                                                                                            | string                | "L2Only" |
```

Comment:
```suggestion
| BPFRedirectToPeer <br/> FELIX_BPFREDIRECTTOPEER                                     | Controls whether it is allowed to forward straight to the peer side of the workload devices, which makes ingress into the workloads from outside the node faster. It is enabled for any host L2 devices by default (`L2Only`), but it breaks tcpdump on the host side of workload device as it bypasses it on ingress. If set it to `Disabled` to make tcpdump work as expected. Value of `Enabled` also allows redirection from L3 host devices like IPIP tunnel or Wireguard directly to the peer side of the workload's device. However, it breaks tools like tcpdump on the peer side. Use `Enabled` with caution.                                                                                                                                                                                                                                                                                            | string                | "L2Only" |
```

`If set it to `Disabled` to make tcpdump work as expected.` Sentence fragment, not sure what's meant here. Can you clarify?